lib/instrumentation/@elastic/elasticsearch.js

```diff
@@ -17,6 +19,15 @@ const logger = require('../../logger').child({ component: 'ElasticSearch' })
  * @returns {void}
  */
 module.exports = function initialize(_agent, elastic, _moduleName, shim) {
+  const pkgVersion = shim.require('./package.json').version
+  if (semver.lt(pkgVersion, '7.13.0')) {
+    shim &&
+      shim.logger.debug(
+        `ElasticSearch support is for versions 7.16.0 and above. Not instrumenting ${pkgVersion}.`
```

Comment:
Shouldn't this be 7.13 or above, based on the condition?

Reply:
Thank you for these suggestions, @villelahdenvuo !